src/main/java/uk/gov/hmcts/reform/migration/ccd/CoreCaseDataService.java

```diff
@@ -49,29 +49,25 @@ public CaseDetails update(String authorisation, String eventId,
 
         CaseDetails updatedCaseDetails = startEventResponse.getCaseDetails();
 
-        if (!updatedCaseDetails.getData().containsKey("caseHandedOffToLegacySite")) {
-            CaseDataContent caseDataContent = CaseDataContent.builder()
-                .eventToken(startEventResponse.getToken())
-                .event(
-                    Event.builder()
-                        .id(startEventResponse.getEventId())
-                        .summary(eventSummary)
-                        .description(eventDescription)
-                        .build()
-                ).data(dataMigrationService.migrate(updatedCaseDetails.getData()))
-                .build();
-            return coreCaseDataApi.submitEventForCaseWorker(
-                AuthUtil.getBearerToken(authorisation),
-                authTokenGenerator.generate(),
-                userDetails.getId(),
-                updatedCaseDetails.getJurisdiction(),
-                caseType,
-                caseId,
-                true,
-                caseDataContent);
-        } else {
-            return null;
-        }
+        CaseDataContent caseDataContent = CaseDataContent.builder()
```

Comment:
should check the applicantOrganisationPolicy & supplementary_data is null before update

Reply:
done